scipy/special/_add_newdocs.py

```diff
@@ -5641,13 +5642,78 @@ def add_newdoc(name, doc):
     The evaluation is carried out using the relation to the incomplete gamma
     integral (regularized gamma function).
 
-    Wrapper for the Cephes [1]_ routine `gdtr`.
+    Wrapper for the Cephes [1]_ routine `gdtr`. Calling `gdtr` directly can
+    improve performance compared to the ``cdf`` method of `scipy.stats.gamma`
+    (see last example below).
 
     References
     ----------
     .. [1] Cephes Mathematical Functions Library,
            http://www.netlib.org/cephes/
 
+    Examples
+    --------
+    Compute the function for `a=1`, `b=2` at `x=5`.
```

Comment:
Use double backticks here and bellow. Should be single tick when only referencing the param yes, but here we show more. So ``` `a` and ``a=1`` ```